src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -191,6 +191,40 @@
                       (read-record add-node-expr-fn)
                       d/cache-node)))}}))
 
+(defn- create-identity-based-handler
+  [clazz
+   tag
+   read-fn
+   write-fn]
+  (let [indexed-tag (str tag "idx")]
```

Comment:
"idx" will be against the tag str, which may be weird to read. Perhaps "-idx" instead?

Reply:
+1, https://github.com/cerner/clara-rules/pull/420/commits/740953759448fbdfafc0a06f0f27e225235bfae0